include/aspect/geometry_model/chunk.h

```diff
@@ -117,6 +117,17 @@ namespace aspect
         virtual
         double longitude_range() const;
 
+
+        virtual
+        double start_latitude() const;
```

Comment:
can you document this and the other similar functions? Would it be worthwhile calling them `south_latitude`, `north_latitude` etc to make it more self-explanatory?